src/test/java/com/box/sdk/BoxSignRequestTest.java

```diff
@@ -0,0 +1,268 @@
+        BoxAPIConnection api = new BoxAPIConnection(TestConfig.getAccessToken());
+
+        String signerEmail = "mwoda+staging@boxdemo.com";
+        List<BoxSignRequestSigner> signers = new ArrayList<BoxSignRequestSigner>();
+        BoxSignRequestSigner newSigner = new BoxSignRequestSigner(signerEmail)
+                .setInPerson(false);
+        signers.add(newSigner);
+
+        String fileId = "11438710730";
+        List<BoxSignRequestFile> files = new ArrayList<BoxSignRequestFile>();
+        BoxSignRequestFile file = new BoxSignRequestFile(fileId);
+        files.add(file);
+
+        String parentFolderId = "6732314306";
+
+        BoxSignRequestCreateParams createParams = new BoxSignRequestCreateParams()
+                .setIsDocumentPreparationNeeded(true);
+        BoxSignRequest.Info signRequestInfo = BoxSignRequest.createSignRequest(api, signers,
+                files, parentFolderId, createParams);
+
+        BoxFile.Info fileInfo = signRequestInfo.getSourceFiles().get(0);
+        BoxSignRequestSigner signer = signRequestInfo.getSigners().get(0);
+
+        Assert.assertNotNull(signRequestInfo.getPrepareUrl());
+        Assert.assertEquals(fileId, fileInfo.getID());
+        Assert.assertEquals(signerEmail, signer.getEmail());
+        Assert.assertNotNull(signRequestInfo.getID());
+    }
+
+    @Test
+    @Category(IntegrationTest.class)
+    public void getSignRequestIntegrationTest() {
+        BoxAPIConnection api = new BoxAPIConnection(TestConfig.getAccessToken());
+
+        final String fileId = "11438710730";
+        final String signerEmail = "mwoda+staging@boxdemo.com";
+        final String signerName = "Mateusz Woda";
+        final String signRequestId = "11446635701-544a1854-c108-4100-9497-4fb7dfd0bcb5";
+
+        BoxSignRequest signRequest = new BoxSignRequest(api, signRequestId);
+        BoxSignRequest.Info signRequestInfo = signRequest.getInfo();
+
+        BoxFile.Info fileInfo = signRequestInfo.getSourceFiles().get(0);
+        BoxSignRequestSigner signer = signRequestInfo.getSigners().get(0);
+
+        Assert.assertEquals(fileId, fileInfo.getID());
+        Assert.assertEquals(signerEmail, signer.getEmail());
+        Assert.assertEquals(signerName, signer.getName());
+        Assert.assertEquals(signRequestId, signRequestInfo.getID());
+    }
+
+    @Test
+    @Category(IntegrationTest.class)
+    public void getSignRequestsIntegrationTest() {
+        BoxAPIConnection api = new BoxAPIConnection(TestConfig.getAccessToken());
+
+        Iterable<BoxSignRequest.Info> signRequests = BoxSignRequest.getAll(api);
+
+        Assert.assertTrue(signRequests.iterator().hasNext());
+    }
+
+    @Test
+    @Category(IntegrationTest.class)
+    public void cancelSignRequestIntegrationTest() throws InterruptedException {
+        BoxAPIConnection api = new BoxAPIConnection(TestConfig.getAccessToken());
+
+        String signerEmail = "mwoda+staging@boxdemo.com";
```

Comment:
Don't merge this information. But, since it's already been compromised (since it's part of this commit), you should change the email address on this account.